ansible/roles-infra/infra-azure-create-inventory/tasks/main.yml

```diff
@@ -29,25 +29,29 @@
     AZURE_CONFIG_DIR: "{{ output_dir }}/.azure-{{project_tag}}"
   changed_when: false
   register: result_list
+  ignore_errors: yes
```

Comment:
The task can fail for another reason.

An error doesn't necessarly mean the resource group doesn't exist.
I think instead you have to look for a 404 + search the response to ensure it doesn't exist.